src/config.test.js

```diff
@@ -33,19 +38,53 @@ describe('setting up config object', () => {
     );
 
     process.env['IOPIPE_CLIENTID'] = 'qux';
+
     expect(setConfig().clientId).toEqual('qux');
 
     // takes IOPIPE_TOKEN over IOPIPE_CLIENTID
     process.env['IOPIPE_TOKEN'] = 'baz';
+
     expect(setConfig().clientId).toEqual('baz');
   });
 
   it('sets timeout windows', () => {
     expect(setConfig({ timeoutWindow: 0 }).timeoutWindow).toEqual(0);
 
     process.env.IOPIPE_TIMEOUT_WINDOW = 100;
+
     expect(setConfig().timeoutWindow).toEqual(100);
     // prefers configuration over environment variables
     expect(setConfig({ timeoutWindow: 0 }).timeoutWindow).toEqual(0);
   });
+
+  it('can be configured via package.json', () => {
```

Comment:
I want to see a test config (json) such that it's easy to see what the config code is modifying, I don't understand how it interacts with plugins.